5.machine_learning/scripts/6.find_sc_crops_coeff.py

```diff
@@ -3,29 +3,29 @@
 
 # # Find one top representative single-cell crop visualize per top healthy and failing coefficients
 # 
-# The features with the highest coefficients per channel per cell type per identified *manually*. We do not use correlation features due to having to use two features and those are harder to visualize.
+# The features with the highest coefficients per channel per cell type are identified *manually*. We do not use correlation features due to having to use two features and those are harder to visualize.
```

Comment:
```suggestion
# The features with the highest coefficients per channel for each cell type are identified manually. We exclude correlation features because they involve two variables, making them more challenging to visualize.
```

Feel free to reject c:

Reply:
I agree with this! Added!